addons/gaea/graph/graph_nodes/root/vector/operations/vector_operation.gd

```diff
@@ -0,0 +1,143 @@
+@tool
+class_name GaeaNodeVectorOp
+extends GaeaNodeVectorBase
+## Base class for operations between 2 vectors.
+
+enum Operation {
+	Add,
+	Subtract,
+	Multiply,
+	Divide,
+}
+
+
+class Definition:
+	var args: Array[StringName]
+	var output: String
+	var conversion: Callable
+	func _init(_args: Array[StringName], _output: String, _conversion: Callable):
+		args = _args
+		output = _output
+		conversion = _conversion
+
+
+var OPERATION_DEFINITIONS: Dictionary[Operation, Definition] : get = _get_operation_definitions
+
+
+func _get_title() -> String:
+	return "VectorOp"
+
+
+func _get_description() -> String:
+	match get_enum_selection(0):
+		Operation.Add:
+			return "Sums 2 [code]Vector[/bg][/c]."
```

Comment:
These should probably match the current type of the node. "Sums 2 [code]Vector2[/bg][/c]s". Same for all the other ones. (Also vector should be plural)

Reply:
Updated